### ISSUE TITLE
[Sage Discussion]: Water Fitting

### ISSUE THREAD

**Issue by ocmadin** [openforcefield/nistdataselection#20]
Water fitting: What water model do we use in this round of fitting? Do we attempt to do any water model parameterization?

**Comment by davidlmobley** [comment]
I think we decided to (a) bring in the data used for TIP3P-FB fitting, and (b) use TIP3P-FB, but (c) NOT fit the water model yet, just include mixtures so that the water model can begin to impact the other parameters.

@leeping is this consistent with your recollection?

**Comment by ocmadin** [comment]
It would be nice to know how dependent the parameterization is on the water model chosen (i.e. if we fit to water mixture data using TIP3P vs. TIP3P-FB, will we end up with drastically different parameter sets?)

**Comment by leeping** [comment]
I agree with Owen and would like to investigate how well mixture properties can be fitted, and how strongly the fitted parameters depend on the choice of water model. My recommendation is to use TIP3P-FB and TIP3P as the two models to focus on. We can include the fitting data used to develop TIP3P-FB (however, I don't see how that data would be useful, as it is pure water data and we are not planning on re-fitting the water model).

**Comment by SimonBoothroyd** [comment]
I agree that the best way forward with this is to begin exploring how the choice of TIP3P or TIP3P-FB affects fitting against aqueous mixture properties, and proceed based upon the results of that.
